app/views/account/pages/rate.js.erb

```diff
@@ -0,0 +1,9 @@
+$('#edit_page_'+'<%= @page.id %>'+' .btn-group').hide();
+$('#edit_page_'+'<%= @page.id %>')
+	.append('<img class="spinner" src=<%= image_path('spinner.gif') %> />');
+setTimeout(function(){
+	$('.spinner').remove();
+	$('#edit_page_'+'<%= @page.id %> .btn-group').show();
+	$('#'+'<%= @page.id %>').html("<%=j render 'rating', { page: @page } %>");
+	$('.progress').parent().closest('div').html("<%=j render 'progress_bar', { page: @page } %>");
```

Comment:
why don't you just add direct HTML tree ID structure to get a direct access to an element instead of traversing through DOM